client/src/components/Task/Task.modules.scss

```diff
@@ -1,12 +1,16 @@
 .task-paper {
-  width: 100%;
   margin: 10px auto;
   padding-top: 0.1rem;
   padding-left: 0.1rem;
   padding-right: 0.1rem;
   position: relative;
 }
 
+.selected-task-paper {
+  box-shadow: 0px 0px 5px 5px rgba($color: #000000, $alpha: 1.0);
```

Comment:
this shadow doesn't seem to be working on my side, perhaps need to add in !important?

Reply:
Oops. I think no need la, since got the double layer highlight

Reply:
haha yea I think so too, it's nice without the shadow, will you remove it?